### PR TITLE
Merge C and K commands

### PR DESCRIPTION
**Pull request by EduardSergeev**
After `aws` removal we need only one set of command

Fixes https://github.com/ambiata/mismi/issues/160

### CODE REVIEW

**Comment by markhibberd** [comment]
:+1:

File: mismi-s3/main/s3.hs [hunk 1]
```diff
@@ -194,6 +124,12 @@ commandA' = subparser $
   <> command' "sync"
               "Sync between two prefixes."
               (Sync <$> address' <*> address' <*> syncMode' <*> fork')
+  <> command' "list"
```
Comment (nhibberd) [inline mismi-s3/main/s3.hs]: Can remove this in favour of `ls` below

File: mismi-s3/main/s3.hs [hunk 2]
```diff
@@ -127,48 +85,20 @@ runC c = do
       getSize a >>= liftIO . maybe exitFailure (putStrLn . show)
     Sync s d m f ->
       syncWithMode m s d f
+    List a rq ->
+      rec (list a) (listRecursively a) rq >>= liftIO . mapM_ (putStrLn . unpack . addressToText)
```
Comment (nhibberd) [inline mismi-s3/main/s3.hs]: Can this be the same as the previous `listRecursively'`implementation, using `list'` and `listRecursively'`. Streaming is very useful :dancers:

**Comment by nhibberd** [comment]
After streaming :boom:

**Pull request merged**